pkg/container-utils/cri/cri.go

```diff
@@ -172,6 +178,17 @@ func parseContainerExtendedData(runtimeName string, containerStatus *pb.Containe
 		},
 	}
 
+	// Fill K8S information.
+	if podName, ok := containerStatus.Labels[containerLabelK8sPodName]; ok {
+		containerExtendedData.PodName = podName
+	}
+	if podNamespace, ok := containerStatus.Labels[containerLabelK8sPodNamespace]; ok {
+		containerExtendedData.PodNamespace = podNamespace
+	}
+	if podUID, ok := containerStatus.Labels[containerLabelK8sPodUID]; ok {
+		containerExtendedData.PodUID = podUID
+	}
```

Comment:
This information is also available in Docker. Please add it also there.

```bash
$ docker ps
CONTAINER ID   IMAGE                  COMMAND                  CREATED         STATUS         PORTS     NAMES
cdacc9c0d32e   busybox                "sleep inf"              7 minutes ago   Up 7 minutes             k8s_test-pod_test-pod_default_eed331a5-c945-4c1d-8daf-5309e90cd2ce_2
...

$ docker inspect cdacc9c0d32e
[
    {
        "Id": "cdacc9c0d32ee333b7fb9b2ce8b4ca82aa3451dfa9ec21d9c5111fa2011cd401",
        ...
        "Config": {
            ...
            "Labels": {
                ...
                "io.kubernetes.container.name": "test-pod",
                "io.kubernetes.docker.type": "container",
                "io.kubernetes.pod.name": "test-pod",
                "io.kubernetes.pod.namespace": "default",
                "io.kubernetes.pod.uid": "eed331a5-c945-4c1d-8daf-5309e90cd2ce",
                "io.kubernetes.sandbox.id": "6c8cf215a8ec3b47201837fccd1e7db674d2a85ec3e9e86cefa8602e5246d44d"
            }
        },
        ...
    }
]

```

Reply:
Added in latest commit